src/h5web/visualizer/Visualizer.test.tsx

```diff
@@ -48,4 +52,24 @@ describe('Visualizer', () => {
       expect(lineTab).toHaveAttribute('aria-selected', 'false');
     });
   });
+
+  test("display error when dataset value can't be fetched", async () => {
+    render(
+      <MockProvider errorOnId="raw">
+        <App />
+      </MockProvider>
+    );
+
+    fireEvent.click(await screen.findByRole('treeitem', { name: 'entities' }));
+
+    const { consoleError, resetConsole } = prepareForConsoleError();
+    fireEvent.click(screen.getByRole('treeitem', { name: 'raw' }));
+
+    expect(await screen.findByText('error')).toBeVisible();
+    expect(consoleError).toHaveBeenCalledTimes(2);
```

Comment:
Why twice ?

Reply:
React calls it once to show to raw stack trace and a second time to show the component stack trace.

Reply:
I've added a comment 👍